org-static-blog.el

```diff
@@ -352,6 +359,15 @@ the variables `org-static-blog-preview-start' and
    "      href=\"" (org-static-blog-get-absolute-url org-static-blog-rss-file) "\"\n"
    "      title=\"RSS feed for " org-static-blog-publish-url "\">\n"
    "<title>" tTitle "</title>\n"
+   "<meta property=\"og:title\" content=\"" tTitle "\">\n"
+   "<meta property=\"og:type\" content=\"article\" />\n"
+   "<meta name=\"twitter:card\" content=\"summary_large_image\">\n"
```

Comment:
what is the `twitter:card` entry?

Reply:
For get about this, I removed it. This is a special tag needed to ensure Twitter/X renders the image.
But, this is something people can manually include in the `header`